modules/Bio/EnsEMBL/Variation/Pipeline/VariationConsequence_conf.pm

```diff
@@ -186,7 +189,7 @@ sub pipeline_analyses {
         ensembl_registry    => $self->o('reg_file'),
         species             => $self->o('species'),
         pipeline_dir => $self->o('pipeline_dir'),
-        max_distance => $self->o('max_distance'),
+        max_distance => ($self->o('species') =~ /homo_sapiens|human/ && ! $self->o('human_default_override')) ? 0 : $self->o('max_distance'),
```

Comment:
Suggest splitting over several lines as long command.
Alos suggest bracketting to make && ! easier to read